.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 Cargo.lock
+/.idea
```

Comment:
This doesn't belong in a per-project `.gitignore`; you might consider adding it to your [global git ignore instead](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
I had no idea that existed, thanks.